docs/reference/api.yml

```diff
@@ -0,0 +1,3204 @@
+                        nutrition:
+                          type: array
+                          items:
+                            type: string
+                    uncharacterized_ingredients_mass_proportion:
+                      type: object
+                      properties:
+                        impact:
+                          type: number
+                        nutrition:
+                          type: number
+                    uncharacterized_ingredients_ratio:
+                      type: object
+                      properties:
+                        impact:
+                          type: number
+                        nutrition:
+                          type: number
+                    warnings:
+                      type: array
+                      items:
+                        type: string
+            ecoscore_extended_data_version:
+              type: string
+            ecoscore_grade:
+              type: string
+            ecoscore_tags:
+              type: array
+              items:
+                type: string
+            editors:
+              type: array
+              items:
+                type: string
+            editors_tags:
+              type: array
+              items:
+                type: string
+            emb_codes:
+              type: string
+            emb_codes_20141016:
+              type: string
+            emb_codes_orig:
+              type: string
+            emb_codes_tags:
+              type: array
+              items:
+                type: object
+            entry_dates_tags:
+              type: array
+              items:
+                type: string
+            expiration_date:
+              type: string
+            food_groups:
+              type: string
+            food_groups_tags:
+              type: array
+              items:
+                type: string
+            fruits-vegetables-nuts_100g_estimate:
+              type: integer
+            generic_name:
+              type: string
+            generic_name_en:
+              type: string
+            id:
+              type: string
+            image_front_small_url:
+              type: string
+            image_front_thumb_url:
+              type: string
+            image_front_url:
+              type: string
+            image_ingredients_small_url:
+              type: string
+            image_ingredients_thumb_url:
+              type: string
+            image_ingredients_url:
+              type: string
+            image_nutrition_small_url:
+              type: string
+            image_nutrition_thumb_url:
+              type: string
+            image_nutrition_url:
+              type: string
+            image_small_url:
+              type: string
+            image_thumb_url:
+              type: string
+            image_url:
+              type: string
+            images:
+              type: object
+              properties:
+                "1":
```

Comment:
Here we have a number of images identified by their id "1", "2" etc. we can have any number of them.

Reply:
We have to look how to document "dynamic" map with json schema, there seems to be support for this.